docs/sphinx/developers/cpp/overview.txt

```diff
@@ -721,7 +721,9 @@ Run :program:`cmake` from the temporary build directory::
 
   % mkdir build
   % cd build
-  % cmake /path/to/bioformats
+  % cmake -G <generator> <options> /path/to/source
```

Comment:
Do you want `[-G <generator>]` as in line 940 (`[-C <configuration>]`) since it isn't mandatory?